src/App.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import Button from '@material-ui/core/Button';
 import IssuesPage from './components/IssuesPage.jsx';
-import UserData from './components/UserData/UserData.jsx'
+import UserData from './components/UserData/UserData.jsx';
```

Comment:
File extensions should not be included in the import unless multiple files in a directory have the same file name.